docs/End-User-Guide.md

```diff
@@ -1,258 +1,430 @@
 
-![Image of the survey output of internal `Matrix Logic`](images/survey-editor-logic-matrix2.png)
+<img src="images/question-types-dynamic-panel.png" alt="Question types - Dynamic panel" width="50%">
 
+### Adorners
 
-## Additional Resources
+Adorners are design-surface controls for survey element manipulation. Use adorners to edit UI texts in place, reorder choices, duplicate and delete elements, specify whether an answer is required, and perform other actions on survey elements. Settings that you specify using adorners are synchronized with settings in the Property Grid, and vice versa.
 
-- [SurveyJS Support Forums](https://surveyjs.answerdesk.io/ticket/list) - useful for searching user questions on specific features
+Each element type has an associated set of adorners. The following image highlights adorners for a [Dropdown](#dropdown) question:
+
+<img src="images/survey-creator-dropdown-adorners.png" alt="Survey Creator - Adorners" width="50%">
+
+### How to: Add a question to the survey
+
+Drag and drop the desired question type from the Toolbox onto the design surface. Alternatively, you can click the **Add Question** button to add a [Single Input](#single-input) question. This button also displays a three-dots icon that allows you to change the type of inserted questions.
+
+<img src="images/survey-creator-add-question-button.png" alt="Survey Creator - Add Question button" width="50%">
+
+### How to: Add a page to the survey
+
+In the Property Grid, select **Survey**, expand the **Pages** category, and click the **Add New** button in the **Pages** editor:
+
+<img src="images/survey-creator-add-new-page.png" alt="Survey Creator - Add new page" width="40%">
+
+Alternatively, you can add a page on the design surface. The surface displays the skeleton of a new page at the bottom. To add a new page, you need to [add a question to it](#how-to-add-a-question-to-the-survey).
+
+### How to: Change the question type
+
+Use the element type selector on the question:
+
+<img src="images/survey-creator-element-type-selector.png" alt="Survey Creator - Element type selector" width="50%">
+
+Some question types are non-interchangeable. For example, a [Dropdown](#dropdown) question supports choices, while [Single Input](#single-input) does not. If you switch the question type from Dropdown to Single Input, your choices will be lost. In this case, you can click the **Undo** button to reinstate the previous configuration:
+
+<img src="images/survey-creator-undo-button.png" alt="Survey Creator - The Undo button" width="50%">
+
+### How to: Clear the survey
+
+Click the **Clear Survey** button on the Toolbar:
+
+<img src="images/survey-creator-clear-survey.png" alt="Survey Creator - The Clear Survey button" width="50%">
+
+## Preview Tab
+
+The Preview tab allows you to see and take the resulting survey as respondents will do it.
```

Comment:
I'd like to simplify this sentence if possible. How about the following?

`The Preview tab allows you to view and take the survey.`

Reply:
It's important that the Preview tab shows the survey as respondents will see it.

Reply:
How about the following? (Note that I don't think "will do it" is necessary at the end.)

`The Preview tab allows you to view and take the survey as a respondent.`